src/design/brand.stories.mdx

```diff
@@ -1,24 +1,15 @@
-import { Meta, IconGallery, IconItem } from '@storybook/addon-docs/blocks';
-import { extname, relative } from 'path';
-// Import whole directory of icons together
-// @see https://webpack.js.org/guides/dependency-management/#context-module-api
-const logoDir = require.context('../assets/brand', true, /\.svg$/);
-const logoElements = logoDir.keys().map((key) => {
-  const Logo = logoDir(key).default;
-  // Retains subdirectory if we want to organize with those in the future
-  const name = relative('.', key).replace(extname(key), '');
-  return (
-    <IconItem name={name}>
-      <Logo class="c-icon" />
-    </IconItem>
-  );
-});
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import brandLogo from './demo/logo.twig';
```

Comment:
Did you try importing `'../assets/brand/logo.svg'` oor `'../assets/brand/logo.svg.twig'` or directly before creating the demo file?

Reply:
No I did not! 

I try that out. Thanks.

Reply:
My commit messages got flip-flopped. Here is what worked for me. 

d29d0b6

Reply:
It looks like my suggestion broke linting. Which makes a bit of sense since Twig files don't get tracked.

If `.svg` (no `.twig`) did not work, then I suppose the demo strategy is fine! But we may want to add a comment like "Must track a Twig template to satisfy CI" or something.

Reply:
Thank you. I missed your comment here, but I came to the same conclusion. 

Since, importing the `.svg` generates an error, I think you suggestion for adding a comment is a great idea. 

![Screen Shot 2020-05-18 at 9 25 13 AM](https://user-images.githubusercontent.com/1427548/82236851-97cfc780-98e9-11ea-80d8-2b703295a7ca.png)

Example of importing `.svg`